pages/image-converter.tsx

```diff
@@ -0,0 +1,195 @@
+  > = (event) => {
+    setFileType(event.currentTarget.value as FileType);
+  };
+  const handleDownload = () => {
+    if (!file) {
+      return;
+    }
+    // Create a canvas of the target size to draw the image
+    const canvas = document.createElement('canvas');
+    const image = new Image();
+    image.src = file.imageSrc;
+    canvas.width = image.width;
+    canvas.height = image.height;
+
+    // Draw the image on the canvas and convert it to the selected type
+    const ctx = canvas.getContext('2d');
+    ctx?.drawImage(image, 0, 0);
+    const dataUrl = canvas.toDataURL(fileType, quality);
+
+    // Create a dummy link to trigger the download
+    const link = document.createElement('a');
+    link.href = dataUrl;
+    link.download = renameFile(file.raw.name, fileType);
+    link.click();
+  };
+  return (
+    <Layout title='Image Converter'>
+      <Heading>Image Converter</Heading>
+      <Container>
+        <Grid
+          container
+          direction={{ xs: 'column', md: 'row' }}
+        >
+          <Grid
+            item
+            xs={6}
+            sx={{
+              height: '500px',
+              borderRadius: '0.5rem',
+              background: 'rgba(0,0,0,.5)',
+              overflow: 'hidden',
+            }}
+          >
+            {file && (
+              <PreviewImage
+                alt=''
+                src={file?.imageSrc}
+                sx={{
+                  width: '100%',
+                  height: '100%',
+                  objectFit: 'contain',
+                }}
+              />
+            )}
+          </Grid>
+          <Grid
+            item
+            xs={6}
+            sx={{ p: 2 }}
+          >
+            <Button
+              variant='contained'
+              component='label'
+            >
+              Upload image
+              <input
+                type='file'
+                accept='image/*'
+                style={{ display: 'none' }}
+                onChange={handleInputChange}
+              />
+            </Button>
+
+            <FormControl sx={{ display: 'flex', my: 3 }}>
+              <FormLabel id='image-type-label'>File Type</FormLabel>
+              <RadioGroup
+                aria-labelledby='image-type-label'
+                name='image-type'
+                value={fileType}
+                onChange={handleFileTypeChange}
+                row
+              >
+                {FILE_TYPE_OPTIONS.map(({ label, value }) => (
+                  <FormControlLabel
+                    key={value}
+                    value={value}
+                    control={<Radio />}
+                    label={label}
+                  />
+                ))}
+              </RadioGroup>
+            </FormControl>
+            <Box sx={{ my: 3 }}>
+              <Typography
+                id='quality-slider'
+                gutterBottom
+              >
+                Quality
+              </Typography>
+              <Slider
```

Comment:
The quality slider doesn't seem to do anything for PNG images, can we disable that input when PNG is selected?